src/content/_Default/build.sh

```diff
@@ -1,3 +1,9 @@
+#if (UsePaket)
+if [ ! -e ".paket/Paket.Restore.targets" ]
+then
+    exec mono .paket/paket.exe install
```

Comment:
Just curious, is there a .NET Core version of Paket which can be run via `dotnet Paket.dll --Install` or something like that? If not it's not a big deal, but if there is a .NET Core version of Paket I think it would be better to use that one instead.

The templates only target .NET Core frameworks and if there's a way to avoid asking users to also have mono installed then I think it would be cleaner, but if Paket only works with mono then the users who would opt in to `--UsePaket` will have it installed anyway, so not a big deal.

Reply:
Still in the works: https://github.com/fsprojects/Paket/pull/2918